controllers/podiochaos/controller.go

```diff
@@ -153,18 +151,17 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		Volume:      obj.Spec.VolumeMountPath,
 		ContainerId: containerID,
 
-		Instance:  obj.Status.Pid,
-		StartTime: obj.Status.StartTime,
-		EnterNS:   true,
+		InstanceUid: obj.Status.Uid,
+		StartTime:   obj.Status.StartTime,
+		EnterNS:     true,
```

Comment:
I think we could also mark these old fields as deprecated in `.proto`.

Or just use "reserved" to remove it directly. 

refer: 
- https://developers.google.com/protocol-buffers/docs/proto3#options, see `deprecated` in section "Options"
- https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
I prefer to use  "reserved".

Reply:
Reserved